src/lib/drupal/staticProps.ts

```diff
@@ -89,7 +89,7 @@ export function getStaticPropsQueryOpts(
   // most types will simply need an ID, preview mode needs context
   const defaultQueryOpts = {
     id,
-    context: context.preview ? context : null,
+    context,
```

Comment:
This function is only used in `getDefaultStaticPropsResource()`, and the return value is (eventually) passed through to `queries.getData()`. This populates the `opts.context` that gets passed through to our data fetcher functions in each query module.

![image](https://github.com/user-attachments/assets/4655b93d-22d6-4ee1-9806-ecb4c6e0f75a)

As you can see, we're not actually using that context anywhere that I can tell, so I think the impact is minimal. (Though to be clear, I don't know why "preview mode needs context" as the comment above says, so there's probably stuff I'm missing. I don't _think_ always providing the context hurts anything, though. 🤞)